platform-modules/mimetype-manager/src/test/scala/org/sunbird/mimetype/mgr/impl/EcmlMimeTypeMgrImplTest.scala

```diff
@@ -80,6 +82,29 @@ class EcmlMimeTypeMgrImplTest extends AsyncFlatSpec with Matchers with AsyncMock
roke\\\":\\\"rgba(255, 255, 255, 0)\\\",\\\"autoplay\\\":false,\\\"visible\\\":true}\"}}]}],\"manifest\":{\"media\":[{\"id\":\"08d97e93-eb90-4970-8e27-49bb171b5895\",\"plugin\":\"org.ekstep.navigation\",\"ver\":\"1.0\",\"src\":\"/content-plugins/org.ekstep.navigation-1.0/renderer/controller/navigation_ctrl.js\",\"type\":\"js\"},{\"id\":\"195c4ae1-5cac-4e4b-88e3-7676c468cd26\",\"plugin\":\"org.ekstep.navigation\",\"ver\":\"1.0\",\"src\":\"/content-plugins/org.ekstep.navigation-1.0/renderer/templates/navigation.html\",\"type\":\"js\"},{\"id\":\"org.ekstep.navigation\",\"plugin\":\"org.ekstep.navigation\",\"ver\":\"1.0\",\"src\":\"/content-plugins/org.ekstep.navigation-1.0/renderer/plugin.js\",\"type\":\"plugin\"},{\"id\":\"org.ekstep.navigation_manifest\",\"plugin\":\"org.ekstep.navigation\",\"ver\":\"1.0\",\"src\":\"/content-plugins/org.ekstep.navigation-1.0/manifest.json\",\"type\":\"json\"},{\"id\":\"d1a56019-00d0-4bbc-a880-5b594eb6db87\",\"plugin\":\"org.ekstep.video\",\"ver\":\"1.5\",\"src\":\"/content-plugins/org.ekstep.video-1.5/renderer/libs/video.js\",\"type\":\"js\"},{\"id\":\"708aa6d2-314e-4d6e-ac53-9c389549cc60\",\"plugin\":\"org.ekstep.video\",\"ver\":\"1.5\",\"src\":\"/content-plugins/org.ekstep.video-1.5/renderer/libs/videoyoutube.js\",\"type\":\"js\"},{\"id\":\"2e2e060d-020d-48e8-aebd-4d8717aa6718\",\"plugin\":\"org.ekstep.video\",\"ver\":\"1.5\",\"src\":\"/content-plugins/org.ekstep.video-1.5/renderer/libs/videojs.css\",\"type\":\"css\"},{\"id\":\"org.ekstep.video\",\"plugin\":\"org.ekstep.video\",\"ver\":\"1.5\",\"src\":\"/content-plugins/org.ekstep.video-1.5/renderer/videoplugin.js\",\"type\":\"plugin\"},{\"id\":\"org.ekstep.video_manifest\",\"plugin\":\"org.ekstep.video\",\"ver\":\"1.5\",\"src\":\"/content-plugins/org.ekstep.video-1.5/manifest.json\",\"type\":\"json\"},{\"id\":\"org.ekstep.text\",\"plugin\":\"org.ekstep.text\",\"ver\":\"1.2\",\"src\":\"/content-plugins/org.ekstep.text-1.2/renderer/supertextplugin.js\",\"type\":\"plugin\"},{\"id\":\"org.ekstep.text_manifest\",\"plugin\":\"org.ekstep.text\",\"ver\":\"1.2\",\"src\":\"/content-plugins/org.ekstep.text-1.2/manifest.json\",\"type\":\"json\"},{\"id\":\"do_2133122975753093121715\",\"src\":\"https://www.youtube.com/watch?v=YH7p6ncEb6g\",\"assetId\":\"do_2133122975753093121715\",\"type\":\"youtube\"},{\"id\":\"do_2133016737715240961616\",\"src\":\"/assets/public/content/do_2133016737715240961616/artifact/do_2133016737715240961616_1623739474410_pexels-photo-594364.jpeg\",\"type\":\"image\"},{\"id\":\"do_2133016742018826241623\",\"src\":\"/assets/public/content/do_2133016742018826241623/artifact/do_2133016742018826241623_1623739526722_.png\",\"type\":\"image\"}]},\"plugin-manifest\":{\"plugin\":[{\"id\":\"org.ekstep.navigation\",\"ver\":\"1.0\",\"type\":\"plugin\",\"depends\":\"\"},{\"id\":\"org.ekstep.video\",\"ver\":\"1.5\",\"type\":\"widget\",\"depends\":\"\"},{\"id\":\"org.ekstep.text\",\"ver\":\"1.2\",\"type\":\"plugin\",\"depends\":\"\"}]},\"compatibilityVersion\":4}}"
```

Comment:
Why don't you use """ for having proper JSON.

Reply:
done